articles/Silo_Finance.md

```diff
@@ -0,0 +1,388 @@
+
+
+## Collateral Risk
+
+In the beginning, the only collateral enabled to borrow XAI will be USDC and ETH. Thus, eliminating other risks that come with long tail assets (e.g., low asset liquidity, high price volatility, depeg of the collateral, etc.).
+
+Both ETH and USDC are highest-tier assets with more than enough liquidity to offset potential liquidations. Even when XAI will be enabled as a bridge asset for more silos, the collateral risk is limited only to those isolated markets.
+
+We currently don’t see any risks related to collateral, however, anyone can propose to add new credit lines. [Credit lines](https://silopedia.silo.finance/welcome/cross-silos-stablecoin) describe the process of allowing silos to use XAI as a bridge asset. While new credit lines need to be approved by governance first, this process can change the composition of XAI’s backing. Potential risks occur in cases where XAI supports illiquid and highly volatile assets. Hence, we recommend that each credit line and silo addition be carefully considered. Silo should also think about installing or incentivizing more detailed risk assessments for each collateral. A bad debt dashboard (as provided by RiskDAO for instance), is another option to better inform users about the health of individual silos.
+
+
+## XAI as Collateral Risk
+
+Silo also aims to enable XAI as collateral within the protocol. As highlighted above, for XAI to become a low-risk collateral, there needs to be enough liquidity in the open market. This will help to facilitate potential liquidations and arbitrage, which is needed to support XAi’s stability.
+
+The team is working on two [initiatives](https://gov.silo.finance/t/building-on-chain-liquidity-for-xai/309) that will ensure the initial provisions of adequate liquidity. First, seeding the initial USDC silo with $1.875M USDC to mint $1.5M XAI. Deposit the minted $1.5M XAI plus an additional $1.5M USDC into a [Uniswap V3 pool](https://info.uniswap.org/#/pools/0x55bb9904df17f3b07551aa117841b3bbfc66646d). The Uni pool is only temporary and will ensure smooth liquidations until the Curve pool XAI/FRAXGP gauge is live.
```

Comment:
Aiham on why they seeded Uni pool instead of Curve pool initially

"Yes, we will move it. We did it because our backup liquidation engine doesn't support curve factory pools yet
Our liquidation engine is our first/last line of defense
There are many liquidators in the ecosystem but we trust none 🙂 - and we are happy to be front-run
we are upgrading the engine soon and once done we will move liquidity to Curve"

He says it will take a few weeks to do this upgrade, after which they will migrate POL

Reply:
adjusted text accordingly:
"The Uni pool is only temporary to ensure smooth liquidations. Once Silo’s own liquidation engine is able to support Curve pools, they will migrate liquidity to the XAI/FRAXBP pool. However, this might take a few weeks to finalize."